src/molecules/BlockHeader/GrayBox/gray-box.module.scss

```diff
@@ -59,8 +59,8 @@
       }
 
       &:before {
-        padding-right: #{$section-padding-desktop};
-        right: -#{$section-padding-desktop};
+        padding-right: #{$section-padding-desktop + $gutter-width};
+        right: -#{$section-padding-desktop + $gutter-width};
```

Comment:
Consider extracting this out into a new variable. 
`$box-offset: $section-padding-desktop + $gutter-width`

Reply:
Sounds good.